controller.js

```diff
@@ -78,3 +79,14 @@ exports.getUsers = (req, res, next) => {
     res.status(200).send({ users });
   });
 };
+
+exports.deleteComment = (req, res, next) => {
+  const id = req.params.comment_id;
+  deleteCommentByCommentId(id)
+    .then(() => {
+      res.status(204).send();
+    })
```

Comment:
Definitely not a necessary refactor, but a fun feature of express is that you can call `res.sendStatus(204)` and it will achieve the same thing as what you're doing here. Just an FYI!